demos/mixer-config-quota-echo.yaml

```diff
@@ -40,17 +75,13 @@ data:
             # we want to increment this counter by 1 for each unique (source, target, service, method, response_code) tuple
             value: "1"
             labels:
-              source: source.service | "unknown"
+              source: source.name | "unknown"
```

Comment:
It's the service the request is being routed to. Per the description on the attribute in this PR, it's "the hostname that was the target of the request."